backend/infrahub/graphql/types/event.py

```diff
@@ -74,6 +76,7 @@ class Meta:
 
     payload = Field(GenericScalar, required=True)
     attributes = Field(List(of_type=NonNull(InfrahubMutatedAttribute), required=True), required=True)
+    related_node = Field(RelatedNode, required=False)
```

Comment:
it feels like we could move that to `EventNodeInterface` since most events will have at least one related node
Also considering that we'll probably need to multiple related nodes, I think it would make sense to use a slightly different name, like `primary_node`

Reply:
I was going back and forth with this a few times, mainly for branch related events but we could potentially add a reference to the branch itself for branch creation events etc. I've moved this to the interface and renamed it `primary_node`.